sites/demo/requirements/base.txt

```diff
@@ -1,9 +1,9 @@
 boto3==1.9.96
```

Comment:
You updated boto3 to `1.16.53` in cnfpt.

Reply:
sorry daddy! done